scripts/Makefile

```diff
@@ -140,7 +140,7 @@ testsuite:
 # It assumes that some covrage reports already exists, otherwise it will fail
 .PHONY: coverage-report
 coverage-report:
-	python ../libbeat/scripts/aggregate_coverage.py -o ./${COVERAGE_DIR}/full.cov ./${COVERAGE_DIR}
+	python ${GOPATH}/src/github.com/elastic/beats/dev-tools/aggregate_coverage.py -o ./${COVERAGE_DIR}/full.cov ./${COVERAGE_DIR}
```

Comment:
We modified the hole thing a little bit. It now always assumes, you have libbeat in the vendor directory including all files. This means the scripts/Makefile is not necessary anymore, as it can link to the original directly.